sso-kit-client/sso-kit-client-react/src/SsoKit.ts

```diff
@@ -0,0 +1,5 @@
+class SsoKit {
+
+}
+
+export const ssoKit = new SsoKit();
```

Comment:
Better focus on the Lit implementation for this PR, and introduce the React one in a separate PR.

Reply:
Removed the React part from this PR. Will create a separate one for that.